src/boot_O2_g3/fault.c

```diff
@@ -40,9 +40,10 @@
  * DPad-Down disables sending fault pages over osSyncPrintf.
  */
 
+#include "prevent_bss_reordering.h"
+#include "prevent_bss_reordering2.h"
```

Comment:
Testing locally these don't seem needed?
Having trouble with the demo ones though. (personally I am fine with the `prevent_bss_reordering2.h` as another stop gap as needed).